File: JavaScript/three.js
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});
function reverse(x) {
    const neg = x < 0;
    let rev = 0;
    x = Math.abs(x);
    while (x > 0) {
        rev = rev * 10 + x % 10;
        x = Math.floor(x / 10);
    }
    return neg ? -rev : rev;
}
async function main() {
    const n = parseInt(await question('Введите количество чисел: '));
    for (let i = 0; i < n; i++) {
        const num = parseInt(await question('Введите число: '));
        console.log(reverse(num));
    }
    rl.close();
}
function question(prompt) {
    return new Promise(resolve => {
        rl.question(prompt, resolve);
    });
}
main();

File: kotlin/three.kt
fun main (){
    print("Введите количество чисел:")
    val n = readLine()!!.toInt()
    for (i in 1..n){
        print("Введите число: ")
        val num = readLine()!!.toInt()
        println("Обратное число: ${reverse(num)}")
    }
}
fun reverse(x:Int):Int{
    val neg = x < 0
    var num = if (neg) -x else x
    var rev = 0
    while (num > 0){
    rev = (rev * 10 + num % 10)
    num = (num / 10)
    }
    return if (neg) -rev else rev
}
